build/luarocks/templates/luarocks_exec.sh

```diff
@@ -59,9 +59,9 @@ variables = {
 }
 " > $ROCKS_CONFIG
 
-LUAROCKS_HOST=$luarocks_host_path
+LUAROCKS_HOST=$luarocks_host_path/../
 
-host_luajit=$root_path/$luajit_path/bin/luajit
+host_luajit=$root_path/$luajit_path/../bin/luajit
```

Comment:
Just wonder if there is any diff between:

```
host_luajit=$root_path/$luajit_path/../bin/luajit

host_luajit=$root_path/bin/luajit
```

Reply:
we have made the way interpolate install location more robust, from simply get the parent directory from first
output to find there common parent folder. And this changes the result for luajit.